frontend/src/main/scala/bloop/engine/BuildLoader.scala

```diff
@@ -217,7 +217,8 @@ object BuildLoader {
     // Recognize 2.12.8-abdcddd as supported if 2.12.8 exists in supported versions
     val isUnsupportedVersion = !supportedScalaVersions.exists(scalaVersion.startsWith(_))
     if (isUnsupportedVersion) {
-      logger.debug(Feedback.skippedUnsupportedScalaMetals(scalaVersion))(DebugFilter.All)
+      if (!scalaVersion.startsWith("3."))
+        logger.debug(Feedback.skippedUnsupportedScalaMetals(scalaVersion))(DebugFilter.All)
```

Comment:
So we still want to enable it in that case? Aha I see that down the line, it is perfectly fine when using Dotty / Scala.

The flag in Bloop is `-Ysemanticdb`, has it been changed to `-Xsemanticdb` in recent versions?

Reply:
It did change, I had to do a quick fix for 3.0.0-M3 :sweat: